### ISSUE TITLE
[ivy] ERROR in Listing specific routes is unsupported for now

### ISSUE THREAD

**Issue by khaledosman** [angular/angular#28690]
# 🐞 bug report

### Affected Package
Not sure, I'm guessing either `@angular/compiler` or `@angular/cli`

### Description

When running the project with `npm start` or `ng serve` An error is thrown complaining about routes. I tried completely removing the router module/routes from my App Module and the error still occurs
```
ERROR in Listing specific routes is unsupported for now (got query for /src/app/app.module#AppModule)
```
After hitting ctrl+s in a ts file to re-trigger the build, the error goes away until the next time I restart the build

## 🔬 Minimal Reproduction
https://github.com/khaledosman/github-search

## 🔥 Exception or Error
<pre><code>
ERROR in Listing specific routes is unsupported for now (got query for /src/app/app.module#AppModule)
</code></pre>

## 🌍  Your Environment
**Angular Version:**
<pre><code>
Angular CLI: 8.0.0-beta.0
Node: 11.9.0
OS: darwin x64
Angular: 8.0.0-beta.3
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router
... service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.14.0-beta.0
@angular-devkit/build-angular     0.14.0-beta.0
@angular-devkit/build-optimizer   0.14.0-beta.0
@angular-devkit/build-webpack     0.14.0-beta.0
@angular-devkit/core              8.0.0-beta.0
@angular-devkit/schematics        8.0.0-beta.0
@angular/cli                      8.0.0-beta.0
@angular/pwa                      0.13.1
@ngtools/webpack                  8.0.0-beta.0
@schematics/angular               8.0.0-beta.0
@schematics/update                0.14.0-beta.0
rxjs                              6.4.0
typescript                        3.3.3
webpack                           4.29.3</code></pre>

Package.json
```json
"dependencies": {
    "@angular/animations": "^8.0.0-beta.3",
    "@angular/common": "^8.0.0-beta.3",
    "@angular/compiler": "^8.0.0-beta.3",
    "@angular/core": "^8.0.0-beta.3",
    "@angular/forms": "^8.0.0-beta.3",
    "@angular/http": "^8.0.0-beta.3",
    "@angular/platform-browser": "^8.0.0-beta.3",
    "@angular/platform-browser-dynamic": "^8.0.0-beta.3",
    "@angular/platform-server": "^8.0.0-beta.3",
    "@angular/pwa": "^0.13.1",
    "@angular/router": "^8.0.0-beta.3",
    "@angular/service-worker": "^8.0.0-beta.3",
    "apollo-angular": "^1.5.0",
    "apollo-angular-link-http": "^1.5.0",
    "apollo-cache-inmemory": "^1.4.2",
    "apollo-client": "^2.4.12",
    "apollo-link": "^1.2.8",
    "apollo-link-context": "^1.0.14",
    "core-js": "^2.6.4",
    "graphql": "^14.1.1",
    "graphql-tag": "^2.10.1",
    "rxjs": "^6.4.0",
    "tslib": "^1.9.3",
    "zone.js": "^0.8.29"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "^0.14.0-beta.0",
    "angular-server-side-configuration": "^1.2.1",
    "@angular/cli": "^8.0.0-beta.0",
    "@angular/compiler-cli": "^8.0.0-beta.3",
    "@angular/language-service": "^8.0.0-beta.3",
    "@types/node": "~11.9.3",
    "husky": "^1.3.1",
    "node-fetch": "^2.3.0",
    "ts-node": "~8.0.2",
    "tslint": "~5.12.1",
    "tslint-config-standard": "^8.0.1",
    "typescript": "~3.3.3"
  }
```

**Comment by alexzuza** [comment]
Seems already covered by https://github.com/angular/angular/commit/e6c51b3e06d9afca416f3b64b987df80444224ac and https://github.com/angular/angular/commit/188f20fb16276863919bfc9bafd41690779003ad

**Comment by gkalpak** [comment]
Thx, @alexzuza! It was indeed fixed in #28542 and it will be included in the next 8.x release.
